phpBB/includes/functions_acp.php

```diff
@@ -229,35 +229,41 @@ function build_select($option_ary, $option_default = false): array
 /**
 * Build radio fields in acp pages
 */
-function h_radio($name, $input_ary, $input_default = false, $id = false, $key = false, $separator = '')
+function build_radio($value, $key, $options)
 {
-	global $user;
+	global $language;
 
-	$html = '';
-	$id_assigned = false;
-	foreach ($input_ary as $value => $title)
+	$buttons = [];
+	foreach ($options as $val => $option)
 	{
-		$selected = ($input_default !== false && $value == $input_default) ? ' checked="checked"' : '';
-		$html .= '<label><input type="radio" name="' . $name . '"' . (($id && !$id_assigned) ? ' id="' . $id . '"' : '') . ' value="' . $value . '"' . $selected . (($key) ? ' accesskey="' . $key . '"' : '') . ' class="radio" /> ' . $user->lang[$title] . '</label>' . $separator;
-		$id_assigned = true;
+		$buttons[] = [
+			'type'		=> 'radio',
+			'value'		=> $val,
+			'name'		=> 'config[' . $key . ']',
+			'checked'	=> $val == $value,
+			'label'		=> $language->lang($option),
+		];
 	}
 
-	return $html;
+	return [
+		'buttons' => $buttons,
+	];
 }
 
 /**
- * HTML-less version of build_cfg_template
+ * Build configuration data arrays or templates for configuration settings
  *
- * @param array $tpl_type Template type
- * @param string $key Config key
- * @param $new_ary
- * @param $config_key
- * @param $vars
- * @return array
+ * @param array		$tpl_type	Configuration setting type data
+ * @param string	$key		Configuration option name
+ * @param array		$new_ary	Updated configuration data
+ * @param string	$config_key	Configuration option name
+ * @param Array		$vars		Configuration setting data
+ *
+ * @return array|string
  */
-function phpbb_build_cfg_template(array $tpl_type, string $key, &$new_ary, $config_key, $vars): array
+function build_cfg_template($tpl_type, $key, &$new_ary, $config_key, $vars)
```

Comment:
Looks like you merge both functions into one? The phpbb_ prefixed one is actually newer and we started to prefix functions a while ago since functions don't have namespaces. If you merge these two, keep the prefix. ;)